ovos_plugin_common_play/ocp/player.py

```diff
@@ -294,6 +294,15 @@ def _get_prefered_audio_backend(self):
         LOG.error("Preferred audio service backend not installed")
         return "simple"
 
+    def _do_gui_audio_play(self):
+        self.bus.emit(Message("gui.player.media.service.play", {
+            "track": self.now_playing.uri,
+            "mime": self.now_playing.mimetype,
+            "repeat": False}))
+        sleep(0.2)  # wait for the above message to be processed
```

Comment:
Could this be `get_response` instead of an arbitrary sleep?

Reply:
there are a few sleeps to be removed still, this PR covers anything >= 1 second only

this one in particular i think is just there to fake some buffering time? feels like it can be removed completely...

will leave for a follow up PR